src/WebUI/src/composables/character/use-inventory-dnd.ts

```diff
@@ -41,7 +40,15 @@ export const useInventoryDnD = (equippedItemsBySlot: Ref<EquippedItemsBySlot>) =
 
   const onDragEnd = (_e: DragEvent | null = null, slot: ItemSlot | null = null) => {
     if (slot && !toSlot.value) {
-      emit('change', [{ userItemId: null, slot }]); // drop outside
+      const items: EquippedItemId[] = [
```

Comment:
Added logic for unequiping related (linked) items
For example, if you unequip the leg armor, you have to unequip the body armor